sparse_strips/vello_cpu/src/render.rs

```diff
@@ -169,6 +168,35 @@ impl RenderContext {
         }
     }
 
+    /// Draw an image into the pixmap. This is a convenience method that is functionally
+    /// equivalent to drawing a rectangle with the same size as the image and applying a pattern fill.
+    ///
+    /// By default, the image will be positioned with the top-left corner at the point (0, 0)
+    /// and the width and height set to the natural dimensions of the image. The positioning and
+    /// scaling of the image can be changed by applying a transform to the render context.
+    pub fn draw_pixmap(&mut self, pixmap: Arc<Pixmap>, image_quality: ImageQuality, alpha: f32) {
```

Comment:
Currently, `vello_hybrid` can’t automatically upload pixmaps (it'd actually panic), but it does work with an `OpaqueId`. From the Unified API perspective, it might be better to use `ImageSource` here instead of a `pixmap`.